pnpm-workspace.yaml

```diff
@@ -6,3 +6,21 @@ packages:
   - 'packages/presets'
   - 'packages/framework/*'
   - 'packages/affine/*'
+
+catalog:
+  '@blocksuite/icons': ^2.1.67
+  '@toeverything/theme': ^1.0.8
+  '@toeverything/y-indexeddb': 0.10.0-canary.9
+
+  '@lit-labs/preact-signals': ^1.0.2
```

Comment:
We plan to use `@preact/signals` directly and use our own lit implementation because the official watcher has a memory leak issue. Can you please also add `@preact/signals` here?